microsoft-edge/webview2/release-notes/index.md

```diff
@@ -88,8 +88,10 @@ For full API compatibility, this Release version of the WebView2 SDK requires We
 The following APIs have been promoted to Stable and are now included in this Release SDK.
 
 
-<!-- ------------------------------ -->
-* Added support for Web Notification, for non-persistent notifications.  The `NotificationReceived` event for `CoreWebView2` controls web notification handling, allowing customization or suppression by the host app.  Unhandled notifications default to WebView2's UI.
+<!-- ---------- -->
+###### Web Notification
```

Comment:
best wording for heading?

see rendered article section:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/release-notes/?branch=pr-en-us-3253&tabs=dotnetcsharp#web-notification

Establishing new, improved formatting/presentation.
@victorhuangwq

Reply:
closing b/c entered Suggested Change comment